### ISSUE TITLE
Final fix phase for code challenges

### ISSUE THREAD

**Issue by jmgasper** [topcoderinc/cab#70]
Hey CAB:  Just thinking through one of the many issues I’ve had recently with running code challenges.  I’d really like to discuss bringing back the idea of a final fix phase to code challenges.  Most of the time, members are very willing to help out and fix issues, but:

1.  I’ve run into situations recently where certain members just refuse to do any work to fix obvious deficiencies in their submissions

2.  I end up having to pay members to do the additional fixes, which isn’t the right incentive to do a good job in the first place, and is a “hidden” cost to the client that I think is a little misleading.

What I’d like to see is a final fix phase tacked onto the end of code challenges.  The final review should be done by the copilot, I think, not a reviewer.  I think it’s important to have the copilot do this independently because they know which items can be skipped and which ones are important to fix.

I think we also need to bring back the old 30 day support period we had with assemblies.  That wasn’t something I used often, but it was nice to have that option, and I think it encourages members to be more supportive of their work.

Discussion on Slack has pointed out that maybe 30 days is too long.  Should we shorten this to 7 or 14 days?

**Comment by dedywahyudi** [comment]
@jmgasper 

+1, Code challenge must have final fixes phase. 
I have situation right now where winner submission not meet all requirements, it's okay if they want to do the fixes. But if they refuse, there's no way to force the winner to do the revision via online review process.

**Comment by lstkz** [comment]
Related ticket #22 

The forum thread from the last year https://apps.topcoder.com/forums/?module=Thread&threadID=889949&start=15&mc=21

There were a few discussions about it on Slack, but everything is deleted now.
